do the tests serially when all is specified to work
around java bug (it looks like)...

Index: tcal-stf.sh
===================================================================
RCS file: /home/icecube/cvsroot/dom-ws/Attic/tcal-stf.sh,v
retrieving revision 1.7.2.1
diff -u -r1.7.2.1 tcal-stf.sh
--- tcal-stf.sh	11 Mar 2005 20:46:34 -0000	1.7.2.1
+++ tcal-stf.sh	5 Apr 2005 18:34:27 -0000
@@ -26,11 +26,14 @@
     fi
     doms=`cat /proc/driver/domhub/card*/pair*/dom*/is-communicating | \
               grep -v NOT | awk '{ print $2 $4 $6; }'`
+    ret=0
     for dom in ${doms}; do
-        $0 ${dom} $2 &
+        if ! $0 ${dom} $2; then
+            echo "`basename $0`: unable to run on dom ${dom}"
+            ret=1
+        fi
     done
-    wait
-    exit $?
+    exit $ret
 fi
 
 card=`echo $1 | awk '{ print substr($0, 1, 1); }'`
